zhaquirks/tuya/ts0601_motion.py

```diff
@@ -426,11 +407,11 @@ class TuyaSensitivityMode(t.enum8):
     )
     .tuya_dp(
         dp_id=102,
-        ep_attribute=TuyaIlluminanceCluster.ep_attribute,
-        attribute_name=TuyaIlluminanceCluster.AttributeDefs.measured_value.name,
+        ep_attribute=TuyaIlluminance.ep_attribute,
+        attribute_name=TuyaIlluminance.AttributeDefs.measured_value.name,
         converter=lambda x: 10000 * math.log10(x) + 1 if x != 0 else 0,
     )
-    .adds(TuyaIlluminanceCluster)
+    .adds(TuyaIlluminance)
```

Comment:
Why can't we use `tuya_illuminance` here (and in the other places below)?
It's the same thing as this, right?

Reply:
Fixed with https://github.com/zigpy/zha-device-handlers/pull/3718/commits/26c01e9aff3f2789cac2588c9bd03e83f0949370